File: rest/call-feedback/instance-get-example-1/instance-get-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

@feedback = @client.account.calls('CAe03b7cd806070d1f32bdb7f1046a41c0')
                   .feedback.fetch

puts @feedback.date_created

File: rest/queue/instance-get-example-1/instance-get-example-1.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

queue = client.queues.get("QU5ef8732a3c49700934481addd5ce1659")
print(queue.average_wait_time)
